app/src/main/java/org/mozilla/fenix/tabtray/TabTrayDialogFragment.kt

```diff
@@ -227,6 +243,31 @@ class TabTrayDialogFragment : AppCompatDialogFragment(), TabTrayInteractor {
         }
     }
 
+    private fun registerCollectionStorageObserver() {
+        requireComponents.core.tabCollectionStorage.register(collectionStorageObserver, this)
+    }
+
+    private fun showCollectionSnackbar() {
+        view.let {
+            val snackbar = FenixSnackbar
+                .make(
+                    duration = FenixSnackbar.LENGTH_LONG,
+                    isDisplayedWithBrowserToolbar = true,
+                    view = (view as View)
+                )
+                .setText(requireContext().getString(R.string.create_collection_tabs_saved))
+                .setAction(requireContext().getString(R.string.create_collection_view)) {
+                    dismissAllowingStateLoss()
+                    if (findNavController().currentDestination?.id != R.id.homeFragment) {
```

Comment:
change this whole block to `.navigate(TabTrayDialogFragmentDirections.actionGlobalHome()` 👍